library/src/scripts/flyouts/hamburgerStyles.ts

```diff
@@ -30,9 +30,6 @@ export const hamburgerClasses = useThemeCache(() => {
     const root = style({});
     const formElVars = formElementsVariables();
     const content = style({
-        ...paddings({
```

Comment:
This seems to cause a little bit of an issue in this list 
![image](https://user-images.githubusercontent.com/1770056/70943460-f46b4580-201e-11ea-8de4-38e8e7b30667.png)

The top & bottom paddings are now misaligned.

Reply:
Fixed